cmd/ak/cmd/sessions/delete.go

```diff
@@ -0,0 +1,50 @@
+package sessions
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/internal/resolver"
+)
+
+var deleteCmd = common.StandardCommand(&cobra.Command{
+	Use:     "delete session ID [--fail]",
+	Short:   "Delete non-running session",
+	Aliases: []string{"d"},
+	Args:    cobra.MaximumNArgs(1),
```

Comment:
use ExactNArgs

Reply:
done

Reply:
@daabr, I have a question regarding the meaning of `--fail`. In many commands we would like to fail/report failure  if the requested resource wasn't found. But it feels that in `delete` we would like to report failure also if the session was in not appropriate for deletion state (e.g. not finished/error). So, I'm in doubts regarding my decision to use this flag to report any error and not only if not found.

Reply:
add FailIfError in addition to FailIfNotFound